ios/RNDeviceInfo/RNDeviceInfo.m

```diff
@@ -488,6 +489,15 @@ - (NSString *) getDeviceTypeName {
     return [DeviceTypeValues objectAtIndex: [self getDeviceType]];
 }
 
+- (int) getBottomNavigationBarHeight {
```

Comment:
If it is constants I'd suppose you can do it in javascript 🤔

Reply:
You are right! Thanks for the feedback!